File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/OrichalcumIngot.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.Item;

/* JADX INFO: loaded from: classes3.dex */
public class OrichalcumIngot extends Item {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.item_orichalcum_ingot_name;
        this.idDescription = R.string.item_orichalcum_ingot_description;
        this.idImage = R.drawable.orichalcum_ingot;
        this.price = 30L;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/OrichalcumIngot.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.Item

class OrichalcumIngot : Item() {
    override fun configureProperties() {
        idName = R.string.item_orichalcum_ingot_name
        idDescription = R.string.item_orichalcum_ingot_description
        idImage = R.drawable.orichalcum_ingot
        price = 30L
    }
}
